src/lib/server/cvpartnerClient.ts

```diff
@@ -76,11 +83,36 @@ export async function fetchCv(userId: string, cvId: string): Promise<Cv> {
 	const ret: Cv = {
 		userid: userId,
 		cvid: cvId,
-		bio: data.key_qualifications[0].long_description.no
+		bio: data.key_qualifications[0].long_description.no,
+		title: data.title.no,
+		name: data.name,
+		profileImage: data.image.url
```

Comment:
Så at du tidligere her fjernet en hardkodet url til et bilde å falle tilbake til hvis `image.url` mangler. Er det noe annet i kulissene nå som fanger opp det tilfellet, eller kan det tenkes å lage trøbbel nå?

Reply:
Nei, det er det ikke, så det kan lage trøbbel hvis noen ikke har lastet opp profilbilde i CvPartner. Vil tro de fleste har det, men kan legge tilbake det hardkodede bilde for safety measures